### PR TITLE
Adds checklist question around China region

### PR DESCRIPTION
**Pull request by dominik-parkopedia**
# Motivation

Less room for errors.

# Description

This PR adds checklist question around china region to our Pull Request template.

## Type of change

- Documentation

# How Has This Been Tested?

n/a

# Checklist

n/a

### CODE REVIEW

File: PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -31,3 +31,6 @@ Are all of the CI checks passing?
 - [ ] I have made corresponding changes to the documentation
 - [ ] I have added tests that prove my fix is effective or that my feature works
 - [ ] New and existing tests pass with my changes
+- [ ] Is the China region affected?
+  - [ ] Are all DB schema changes synchronised?
+  - [ ] Does code handle edge cases correctly?
```
Comment (AshleySheridanPP) [inline PULL_REQUEST_TEMPLATE.md]: I think for this, perhaps changing the wording to something like "Is the code correctly handling the setup in the China region?"

Comment (dominik-parkopedia) [reply PULL_REQUEST_TEMPLATE.md]: Good point, that's more precise!

Comment (dominik-parkopedia) [reply PULL_REQUEST_TEMPLATE.md]: Done!

**Review by GaneshParkopedia — APPROVED** [review]

**Review by AshleySheridanPP — APPROVED** [review]

**Comment by GaneshParkopedia** [comment]
Going to merge this for Dom as he is on holiday
